twa-matching.md

```diff
@@ -0,0 +1,24 @@
+# Project TWA Careers and Candidates Matching
+
+## Client Profile
+
+The Transformative Workforce Academy (TWA) is an initiative at Saint Louis University aimed at supporting individuals at risk of incarceration to lead productive lives without returning to prison. SLU's comprehensive approach includes prevention, in-prison, and reentry services. TWA specifically focuses on helping individuals leaving prison to find and maintain meaningful employment.
+
+TWA started its efforts in 2018 by organizing the Second Chance Job Fair in partnership with federal and state probation and parole offices. The fair aimed to connect job seekers with employers who consider candidates holistically, beyond their criminal records. Building on the success of these job fairs, TWA now aims to provide year-round support to jobseekers and employers through the Second Chance Job Portal. The Second Chance Job Portal will offer resources and assistance for jobseekers seeking second-chance employment, as well as support for employers interested in hiring and retaining this often overlooked segment of the workforce. The goal is to facilitate successful employment opportunities and help address the root causes of crime by providing ongoing support and resources to those seeking a second chance.
+
+## Project Brief
+
+This project involves collecting and matching the below information
+
+1. criminal records of individuals
+2. Job requirements from the hiring team.
+
+If the above 1 and 2 requirements are not related to each other. then we can send this particular profile to hiring team. similarly vice versa.
+
+## Functional Requirements
+
+* Profile building.
+* Job posting.
+* Apply for jobs.
+* Filtering candidates.
+* Training candidates.
```

Comment:
Here we can add one more functional requirement as 

Application Monitoring:

->Employers should be able to use an application tracking system to manage received applications, follow candidate progress, and communicate application status.
->Allow job seekers to check the status of their applications and receive notifications about changes.

Reply:
Would this be redundant to the ATS systems each employer already has? I don't think we want to build something to replace their systems for hiring. My understanding is that we are responsible for building a separate system that is primarily about enabling the matching done by the TWA program. Maybe I'm misunderstanding your suggestions. Can you clarify?

Reply:
Thank you for your input and explanation. You are correct that the major goal of project is to facilitate the TWA program's matching procedure. The proposed functional need of 'Application Monitoring' intends to provide a feature inside our system that allows companies to track and manage applications received via the TWA platform, hence giving transparency and allowing effective communication between employers and job seekers. While many organizations may already have Applicant Tracking Systems (ATS) in place for managing applications, our suggested feature is not meant to replace those systems. Instead, it complements our platform by providing an easy mechanism for employers to monitor TWA- specific applications and guarantee the recruiting process for TWA prospects is handled efficiently. 

We hope to improve the overall user experience by providing an application tracking system within our platform for employers participating in the TWA program. So, this is my suggestion, does this clarify your question? please let me know if any.

Reply:
thank you for clarification.